packages/icons/package.json

```diff
@@ -16,16 +16,19 @@
     "prepare:react": "node ./scripts/prepareSvgReact.js",
     "build:svg": "node ./scripts/buildSvg.js",
     "build:clean": "shx rm -rf dist",
-    "build:prepare": "shx mkdir -p dist/svg dist/react dist/.tmp-svg && shx cp -r package.json README.md dist/",
+    "build:prepare": "shx mkdir -p dist/svg dist/react dist/.tmp-svg dist/constants",
+    "build:copy": "shx cp -r package.json README.md dist/ && shx cp -r src/react/* dist/react",
     "build:react": "svg2react-icon --typescript --no-sub-dir ./dist/.tmp-svg ./dist/react",
     "build:react:clean": "shx rm -rf dist/react/**.d.ts dist/react/**.js.map dist/.tmp-svg",
+    "build:const": "node ./scripts/buildConstants.js",
     "prebuild": "npm-run-all build:clean build:prepare",
-    "build": "npm-run-all build:svg prepare:react build:react",
-    "postbuild": "yarn build:react:clean"
+    "build": "npm-run-all build:svg prepare:react build:react build:const",
+    "postbuild": "npm-run-all build:react:clean build:copy"
   },
   "devDependencies": {
+    "jsdom": "^20.0.0",
```

Comment:
jsdom to dependencies (not devD)?

Reply:
JSDOM is used only for generating icons, so it is a devDependency.